package/Dockerfile

```
@@ -43,25 +43,25 @@ RUN curl --output /tmp/kustomize.tar.gz -sLf "https://github.com/kubernetes-sigs
     echo "${!KUSTOMIZE_SUM}  /tmp/kustomize.tar.gz" | sha256sum -c - && \
     tar -xvzf /tmp/kustomize.tar.gz -C / kustomize
 
-# Stage k9s into build
-ENV K9S_SUM="K9S_SUM_${TARGETARCH}"
-RUN curl --output /tmp/k9s.tar.gz -sLf "https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${TARGETARCH}.tar.gz" && \
-    echo "${!K9S_SUM}  /tmp/k9s.tar.gz" | sha256sum -c - && \
-    tar -xvzf /tmp/k9s.tar.gz -C / k9s
-
 FROM registry.suse.com/bci/bci-base:${BCI_VERSION} AS zypper
 
 # Creates the based dir for the target image, and hydrates it with the
 # original contents of the final image.
 RUN mkdir /chroot
 COPY --from=final / /chroot/
 
+# Setup repo needed to install k9s package
+RUN zypper --non-interactive ar https://download.opensuse.org/repositories/devel:kubic/15.6/devel:kubic.repo \
+    && zypper --non-interactive ar https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Factory_ARM/ devel_kubic_arm \
+    && zypper --non-interactive ar https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Factory_zSystems/ devel_kubic_zed \
+    && zypper --non-interactive --gpg-auto-import-keys refresh
```

Auto import keys is better avoided. 

```suggestion
rpm --import https://download.opensuse.org/repositories/devel:/kubic/15.6/repodata/repomd.xml.key && \
    && zypper --non-interactive refresh
```